tests/snapshot_tests/test_snapshots.py

```diff
@@ -256,3 +256,7 @@ def test_disabled_widgets(snap_compare):
 
 def test_focus_component_class(snap_compare):
     assert snap_compare(SNAPSHOT_APPS_DIR / "focus_component_class.py", press=["tab"])
+
+
+def test_line_api_scrollbars(snap_compare):
+    assert snap_compare(SNAPSHOT_APPS_DIR / "test_line_api_scrollbars.py", press=["_"])
```

Comment:
The app examples in `snapshot_apps` don't have the leading `test_` in their names.
I don't think this will break anything, but it may create the expectation (for humans and for `pytest`) that there are tests written in this file.